src/load_order.ts

```diff
@@ -681,6 +855,22 @@ const deployAndSerializeNewLoadOrder: VortexWrappedSerializeFunc = (
   const serializedLoadOrder =
     encodeLoadOrder(v2077LoadOrder);
 
+  const maybeRebuiltModsDotJson: TaskEither<Error, string> = pipe(
```

Comment:
Think this needs to wait on the deployment we kick off above to complete, right?

The LO disk write was here outside the async deployment as a fallback if Vortex crashes or something - but tbh I'm not sure how useful or needful that is, so if it's easiest to just put the whole chain in the async world, I think it should be fine.

Reply:
mhmm, I will take a look at that then